src/com_tjnotifications/admin/config.xml

```diff
@@ -6,6 +6,10 @@
 			<option value="1">JYES</option>
 		</field>
 
+		<field name="tjmail_plugin" type="sql" default="" label="COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_EMAIL_PROVIDER" description="COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_EMAIL_PROVIDER_DESC" query="SELECT element, name FROM #__extensions WHERE type='plugin' AND folder='tjmail' AND enabled=1" key_field="element" value_field="name">
```

Comment:
@tushar33 Is the code to use the selected plugin in a different MR?

Reply:
That code is in tjnotification library
https://git.tekdi.net/techjoomla/lib_techjoomla/-/merge_requests/154